cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -118,7 +118,7 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Distinct",
       text =
-        """Removes duplicate rows from the incoming stream of rows.""".stripMargin,
+        """The `Distinct` operator removes duplicate rows from the incoming stream of rows.""".stripMargin,
```

Comment:
This operator needs to build up state to make sure only distinct elements are returned. This leads to added memory pressure in the system.

Reply:
👍